### PR TITLE
Slait subscriber plugin

### PR DESCRIPTION
**Pull request by rocketbitz**

### CODE REVIEW

File: contrib/slait/glide.lock [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  version: 9a2d84e8319e451f0145d3d6c10467cd6821540f
+  subpackages:
+  - cache
+  - commitlog
+  - rest
+  - rest/client
+  - socket
+  - utils
+  - utils/log
+- name: github.com/alpacahq/marketstore
```
Comment (umitanuki) [inline contrib/slait/glide.lock]: Did you need to add this?  I think it's going be problem since a plugin has to build with exact server version

Comment (rocketbitz) [reply contrib/slait/glide.lock]: ya, glide pulled it in. I removed.

File: contrib/slait/slait.go [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+	}()
+	for {
+		msgType, msg, err := ss.conn.ReadMessage()
+		if err != nil {
+			glog.Errorf("Failed to read message from Slait - Error: %v", err)
+			return err
+		}
+		err = ss.handleMessage(msg, msgType)
+		if err != nil {
+			return err
```
Comment (umitanuki) [inline contrib/slait/slait.go]: This error is not handled at all

File: contrib/slait/slait.go [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+	go ss.read()
+
+	// subscribe to all symbols on the partition
+	subMsg := socket.SocketMessage{
+		Action: "subscribe",
+		Topic:  ss.partition,
+	}
+	err = ss.conn.WriteJSON(subMsg)
+	if err != nil {
+		return ss.reconnect(5 * time.Second)
```
Comment (umitanuki) [inline contrib/slait/slait.go]: It should tell the dispatched goroutine to stop here?

File: contrib/slait/slait.go (hunk 2 above)
Comment (rocketbitz) [reply contrib/slait/slait.go]: ya missed a log there.

File: contrib/slait/slait.go (hunk 3 above)
Comment (rocketbitz) [reply contrib/slait/slait.go]: yes. closing the conn will do the trick.

Comment (rocketbitz) [reply contrib/slait/slait.go]: which is done in the defer above.

**Pull request merged**